### ISSUE TITLE
Captcha Challenge

### ISSUE THREAD

**Issue by thangckt** [Integration-Automation/ReEdgeGPT#157]
hi @JE-Chen 

I get this error
```
ResponseError: CaptchaChallenge: User needs to solve CAPTCHA to continue. 
```

Do you know how to solve it?

Thank.

**Comment by JE-Chen** [comment]
![image](https://github.com/Integration-Automation/ReEdgeGPT/assets/33644111/277ff937-31a2-48d1-aa4e-9e55043b4df5)
![image](https://github.com/Integration-Automation/ReEdgeGPT/assets/33644111/0a47c0f3-a442-46dc-9f35-f8cae23e9f0d)
![image](https://github.com/Integration-Automation/ReEdgeGPT/assets/33644111/3dc542ca-c3f0-454c-b61c-be0bbcac54f3)

Manual send message to Copilot and waiting Copilot verify, If Copilot reply, Export new cookie file and use new cookie file.
![image](https://github.com/Integration-Automation/ReEdgeGPT/assets/33644111/ce511e87-c373-485c-9483-f4eba4670562)

**Comment by thangckt** [comment]
this work, thank
@JE-Chen, do you have plan to reverse openai's web?

**Comment by JE-Chen** [comment]
Already have swagger doc and flask example.

Swagger doc:
https://github.com/Integration-Automation/ReEdgeGPT/blob/main/Dockerfiles/Flask/docs/static/openapi.yaml

Flask example:
https://github.com/Integration-Automation/ReEdgeGPT/tree/main/Dockerfiles/Flask

You can copy all raw doc text and paste text on new official swagger editor:
https://editor-next.swagger.io/

Also, you can build your own docker container.

**Comment by thangckt** [comment]
hi @JE-Chen 
Thank you for your information.

I mean, do you have plan to do a reverse engineering website chatgpt.com, like the way you are doing with microsoft copilot here.
Thank.

**Comment by JE-Chen** [comment]
> hi @JE-Chen Thank you for your information.
> 
> I mean, do you have plan to do a reverse engineering website chatgpt.com, like the way you are doing with microsoft copilot here. Thank.

No, there are too many open-source projects that do this

**Comment by thangckt** [comment]
> No, there are too many open-source projects that do this
Yes, but they may not really work, or inefficient. I really like your work in this repo, and hope to see a similar work for chatgpt.

Thank.
